src/components/chat/chat-panel.tsx

```diff
@@ -7,7 +7,9 @@ import {
 } from "../../hooks/document-comment-hooks";
 import { useDeleteDocument } from "../../hooks/firestore-hooks";
 import { useDocumentOrCurriculumMetadata } from "../../hooks/use-stores";
+
 import "./chat-panel.scss";
+import { isDocumentMetadata } from "../../../functions/src/shared";
```

Comment:
We tend to put our style imports at the end of the import list. So `import "./chat-panel.scss";` would be the last import in this module.

Reply:
Done